styles/config.scss

```
@@ -35,10 +35,10 @@
   
     /* Litir */
     /** Litir með nafni **/
-    --color-yellow: #f0f;
+    --color-yellow: #aa6644;
```

Held að þetta sé betri litur: 
```suggestion
    --color-yellow: #bb6644;
```